contracts/test/SideTokenFactoryFlatten.sol

```diff
@@ -0,0 +1,1442 @@
+// Sources flattened with hardhat v2.6.1 https://hardhat.org
```

Comment:
I think we can change this for a mock

Reply:
It`s better than using the flatten ?

Reply:
it's easier to follow, as the flatten has a lot of stuff, let's discuss it on the meeting we have for the other stuff